module/renderer/src/commonTest/kotlin/org/cru/godtools/shared/renderer/content/RenderTabsTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
+import androidx.compose.ui.test.onNodeWithText
+import org.ccci.gto.support.androidx.test.junit.runners.AndroidJUnit4
+import org.ccci.gto.support.androidx.test.junit.runners.RunOnAndroidWith
+import org.cru.godtools.shared.tool.parser.model.Tabs
+
+@RunOnAndroidWith(AndroidJUnit4::class)
+class RenderTabsTest : BaseRenderContentTest() {
+    override val testModel = Tabs(
+        tabs = listOf(
+            Tabs.Tab(label = null),
+            Tabs.Tab(label = null)
+        ),
+        invisibleIf = invisibleIf,
+        goneIf = goneIf
+    )
+    override fun SemanticsNodeInteractionsProvider.onModelNode() = onNodeWithText("Test")
```

Comment:
the semantics matcher needs to match the `Tabs` element, right now it's looking for a text node with "Test". take a look at some of the other test matchers for `onNodeWithTag` to see how to find the node you are looking for